### DISCUSSION TITLE
[Doc] Repurpose the Kubernetes chapter with IA improvements

### DISCUSSION THREAD

**Discussion by momo-jun** [apache/pulsar#17866] [General]
In the docs,  there are many categories in the left navigation that do not make sense and may distract the focus, and some need IA improvements too.

Let's start with [`Kubernetes (Helm)`](https://pulsar.apache.org/docs/next/helm-overview) to get a quick win - actually, it's not a feature set of Pulsar and way more related to the deployment. I think it can be repurposed and incorporated into the `Deployment` category with a lean structure. Many topics inside also have duplicate or inappropriate content chunks, for example, the major content of the `Install` and `Required tools` topics can be put in the `Prepare` topic.

My initial thought on the content mapping is as follows. 
<img width="528" alt="image" src="https://user-images.githubusercontent.com/60642177/192712868-ae2cdfe8-c9e5-478b-a15e-e744eeab9f0a.png">

Feel free to leave your comments and let me know whether it makes sense to you.
//cc @D-2-Ed @DaveDuggins @Anonymitaet

**Comment by tisonkun** [comment]
Related issue: https://github.com/apache/pulsar/issues/17774.

**Comment by D-2-Ed** [comment]
@momo-jun - I agree there are many distracting categories in the left navigation. The IA calls for quickstart section and deployment section under Pulsar A-Z. Please confer with @DaveDuggins  who worked on this content. 
K8s belongs in installation/deployment and it is 1 of 3 ways to get up and running with pulsar. 
From a task perspective, be sure we are explaining what is important to decision making for Pulsar and the xref to the K8s (3rd party) documentation for more detailed information.
